WoloxAndroidBootstrp/app/src/main/java/ar/com/wolox/android/example/ui/example/ExampleFragment.kt

```diff
@@ -5,23 +5,47 @@ import ar.com.wolox.android.R
 import ar.com.wolox.android.databinding.FragmentExampleBinding
 import ar.com.wolox.android.example.ui.viewpager.ViewPagerActivity
 import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import ar.com.wolox.wolmo.core.util.ToastFactory
 import ar.com.wolox.wolmo.core.util.openBrowser
 import ar.com.wolox.wolmo.core.util.openDial
+import javax.inject.Inject
 
 class ExampleFragment private constructor() : WolmoFragment<FragmentExampleBinding, ExamplePresenter>(), ExampleView {
 
+    @Inject
+    internal lateinit var toastFactory: ToastFactory
+
     override fun layout() = R.layout.fragment_example
 
     override fun init() {
     }
 
     override fun setListeners() {
         with(binding) {
-            usernameInput.addTextChangedListener { presenter.onUsernameInputChanged(it.toString()) }
+            etEmail.addTextChangedListener { presenter.onUsernameInputChanged(it.toString()) }
             woloxLink.setOnClickListener { presenter.onWoloxLinkClicked() }
             woloxPhone.setOnClickListener { presenter.onWoloxPhoneClicked() }
             loginButton.setOnClickListener {
-                presenter.onLoginButtonClicked(usernameInput.text.toString(), favouriteColorInput.text.toString())
+                val email: String = binding.etEmail.text.toString()
+                val password: String = binding.etPassword.text.toString()
```

Comment:
You don't need to call binding here, since you are inside a with(binding) block.